### ISSUE TITLE
Refactor magic numbers for weights and measures

### ISSUE THREAD

**Issue by andrewpbrett** [openfoodfoundation/openfoodnetwork#7046]
### What we should change and why (this is tech debt)

We have magic numbers (e.g. 453.6 for the conversion to pounds from grams) in a lot of different places in the code. We should have them in one or two places (front and backend) at most. 

### Context

https://github.com/openfoodfoundation/openfoodnetwork/pull/6956#discussion_r589472683

**Comment by jibees** [comment]
I agree. I've added some under the specs too : fbcdd6bdb435a2d58e2d9416ffc1d7ce78e2c9d3
